graylog2-web-interface/src/components/lookup-tables/LookupTablesOverview.tsx

```diff
@@ -168,27 +172,29 @@ const LookupTablesOverview = ({
             </OverlayTrigger>
           </SearchForm>
           <div style={{ overflowX: 'auto' }}>
```

Comment:
Thanks for working on this. Code LGTM.

Since we want to avoid `inline styles` It will be great if you can use `styled-components` here.

Reply:
Thank you for spotting this. I fixed it!